### DISCUSSION TITLE
Is there anyway to add custom fonts to the Console in the web version of cloudshell?

### DISCUSSION THREAD

**Discussion by brwilkinson** [Azure/CloudShell#135] [Q&A] [answered]
I would like a font that supports ligatures? E.g. a nerdfont.

- Ideally if there was 1 of the defaults, or if there was a way to import?

![image](https://user-images.githubusercontent.com/3605266/149418743-7fe3bada-bc63-4361-84ec-39fa25603f57.png)

**Comment by edyoung** [comment, marked as answer]
Sadly not at present but it's on the TODO list. Might be possible with a browser extension in the meantime.

**Reply by brwilkinson** [reply]
Okay, thanks, I saw another thread on the CSS settings, so I'll take a look over there.
